### PR TITLE
Topic 01

### PR DESCRIPTION
**Pull request by mateocarles**

### CODE REVIEW

File: mateo/00-HTML-CSS-basics/htmlej1_2_3.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (marcelososa) [inline mateo/00-HTML-CSS-basics/htmlej1_2_3.html]: Please, move these changes to another PR related to Topic 0.

File: mateo/00-HTML-CSS-basics/htmlej4.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```
Comment (marcelososa) [inline mateo/00-HTML-CSS-basics/htmlej4.html]: Please, move these changes to another PR related to Topic 0.

File: mateo/00-HTML-CSS-basics/htmlej5.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (marcelososa) [inline mateo/00-HTML-CSS-basics/htmlej5.html]: Please, move these changes to another PR related to Topic 0.

File: mateo/00-HTML-CSS-basics/htmlej6.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```
Comment (marcelososa) [inline mateo/00-HTML-CSS-basics/htmlej6.html]: Please, move these changes to another PR related to Topic 0.

File: mateo/00-HTML-CSS-basics/index.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
+<!doctype html>
```
Comment (marcelososa) [inline mateo/00-HTML-CSS-basics/index.html]: Please, move these changes to another PR related to Topic 0.

File: mateo/00-HTML-CSS-basics/layout.css [hunk 6]
```diff
@@ -0,0 +1,504 @@
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
```
Comment (marcelososa) [inline mateo/00-HTML-CSS-basics/layout.css]: Please, move these changes to another PR related to Topic 0.

File: mateo/00-HTML-CSS-basics/layout.html [hunk 7]
```diff
@@ -0,0 +1,46 @@
+<!doctype html>
```
Comment (marcelososa) [inline mateo/00-HTML-CSS-basics/layout.html]: Please, move these changes to another PR related to Topic 0.

File: mateo/00-HTML-CSS-basics/normalize.css [hunk 8]
```diff
@@ -0,0 +1,537 @@
+/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```
Comment (marcelososa) [inline mateo/00-HTML-CSS-basics/normalize.css]: Please, move these changes to another PR related to Topic 0.

File: mateo/00-HTML-CSS-basics/style.css [hunk 9]
```diff
@@ -0,0 +1,43 @@
+html, body, div, span, applet, object, iframe,
```
Comment (marcelososa) [inline mateo/00-HTML-CSS-basics/style.css]: Please, move these changes to another PR related to Topic 0.

File: mateo/01-JS-DOM-APIs/index.html [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+        </div>
+   
+
+        <ul id="list"></ul>
+
+      
+ 		
+
+        <script type="text/javascript">
+
```
Comment (marcelososa) [inline mateo/01-JS-DOM-APIs/index.html]: Move this javascript block inside your js file.

File: mateo/01-JS-DOM-APIs/index.html [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+        <ul id="list"></ul>
+
+      
+ 		
+
+        <script type="text/javascript">
+           
+           fadeIn('fadeSection');
+
+            document.getElementById("btn").onclick = function(){
```
Comment (marcelososa) [inline mateo/01-JS-DOM-APIs/index.html]: You don't need an anonym function to call the `btnclk` function, you can do something like that:
```javascript
document.getElementById("btn").onclick = btnclck;
```

File: mateo/01-JS-DOM-APIs/js.js [hunk 12]
```diff
@@ -0,0 +1,167 @@
+var fade_in_from = 0;
```
Comment (marcelososa) [inline mateo/01-JS-DOM-APIs/js.js]: Is not good idea declare global variables for a specific feature. What if you need the same effect for another elements?
Maybe you can use a recursive function to do that, just an idea.

File: mateo/01-JS-DOM-APIs/js.js [hunk 13]
```diff
@@ -0,0 +1,167 @@
...
+
+//*ex nº 1/2/3/4
+
+function fadeIn(element){
+
+     var target = document.getElementById(element);
+     target.style.display = "block";
```
Comment (marcelososa) [inline mateo/01-JS-DOM-APIs/js.js]: Styles should be handled by CSS classes.

File: mateo/01-JS-DOM-APIs/js.js [hunk 14]
```diff
@@ -0,0 +1,167 @@
...
+            
+     var newSetting = fade_in_from / 10;
+     target.style.opacity = newSetting;
```
Comment (marcelososa) [inline mateo/01-JS-DOM-APIs/js.js]: Styles should be handled by CSS classes.

File: mateo/01-JS-DOM-APIs/js.js [hunk 15]
```diff
@@ -0,0 +1,167 @@
...
+
+      fade_in_from++;
+
+     if(fade_in_from==10){
+     target.style.opacity= 1;
```
Comment (marcelososa) [inline mateo/01-JS-DOM-APIs/js.js]: Styles should be handled by CSS classes.

File: mateo/01-JS-DOM-APIs/js.js [hunk 16]
```diff
@@ -0,0 +1,167 @@
...
+        request:"GET",
+        send:null
+    }
+    get(config)
+    .then(function(j){
+                
+                document.getElementById("list").style.opacity = 1;
+            
+                
+                 for(i = 0 ; i < j.items.length ; i++){
```
Comment (marcelososa) [inline mateo/01-JS-DOM-APIs/js.js]: This variable will be declared globally if you don't define its scope.

File: mateo/01-JS-DOM-APIs/js.js [hunk 17]
```diff
@@ -0,0 +1,167 @@
...
+        request:"GET",
+        send:null
+    }
+    get(config)
+    .then(function(j){
+                
+                
+            
+                
+                 for(i = 0 ; i < j.items.length ; i++){
```
Comment (marcelososa) [inline mateo/01-JS-DOM-APIs/js.js]: This variable will be declared globally if you don't define its scope.

File: mateo/01-JS-DOM-APIs/js.js (hunk 12 above)
Comment (mateocarles) [inline mateo/01-JS-DOM-APIs/js.js]: I've found a much better way to do the fade in

**Comment by marcelososa** [comment]
LGTM 👍
